packages/typespec-test/test/hierarchy_generic/generated/typespec-ts/review/hierarchy-generic.api.md

```diff
@@ -9,31 +9,31 @@ import { OperationOptions } from '@azure-rest/core-client';
 import { Pipeline } from '@azure/core-rest-pipeline';
 
 // @public (undocumented)
-export interface A {
+export interface A0 {
```

Comment:
haha, we start with _0? i was wondering if we just keep A as A for the first one?